Mathlib/NumberTheory/Harmonic/GammaDeriv.lean

```diff
@@ -79,49 +81,151 @@ lemma deriv_Gamma_nat (n : ℕ) :
 
 
 lemma hasDerivAt_Gamma_nat (n : ℕ) :
-    HasDerivAt Gamma ((n)! * (-eulerMascheroniConstant + harmonic n)) (n + 1) :=
+    HasDerivAt Gamma ((n)! * (-γ + harmonic n)) (n + 1) :=
   (deriv_Gamma_nat n).symm ▸
     (differentiableAt_Gamma fun m ↦ (by linarith : (n : ℝ) + 1 ≠ -m)).hasDerivAt
 
-lemma eulerMascheroniConstant_eq_neg_deriv : eulerMascheroniConstant = -deriv Gamma 1 := by
+lemma γ_eq_neg_deriv : γ = -deriv Gamma 1 := by
```

Comment:
Can you add a deprecated alias for the old name? Just in case somebody has been using it...

Reply:
Oops, the change was unintentional, I'll revert it.